dapps/templates/simple/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "embark-dapp-template-simple",
+  "version": "4.0.0-beta.0",
+  "description": "",
+  "files": [
+    "contracts",
+    "contracts.js",
+    "dot.gitignore",
```

Comment:
Is there a particular reason for prefixing these files with `dot.` instead of just using `.`?

Reply:
`.gitignore` gets excluded automatically by npm when the package is published, so `dot.gitignore` is a way of preserving the one we want in the instantiated template. The template generator renames the copied file `.gitignore` during instantiation. `dot.npmignore` has a slightly different purpose — the `.npmignore` in the template is suited to the monorepo, while the copied `dot.npmignore` gets renamed to `.npmignore`.